ui/packages/components/src/TimelineV2/TraceHeading.tsx

```diff
@@ -92,12 +93,15 @@ function TimeWithText({ trace }: { trace: Props['trace'] }) {
   let value: Date;
   if (trace.endedAt) {
     text = 'Ended';
-    if (trace.status === 'CANCELLED') {
-      text = 'Cancelled';
-    } else if (trace.status === 'SUCCEEDED') {
-      text = 'Completed';
-    } else if (trace.status === 'FAILED') {
-      text = 'Failed';
+
+    if (isFunctionRunStatus(trace.status)) {
+      if (trace.status === 'CANCELLED') {
+        text = 'Cancelled';
+      } else if (trace.status === 'COMPLETED') {
+        text = 'Completed';
+      } else if (trace.status === 'FAILED') {
+        text = 'Failed';
+      }
```

Comment:
**suggestion (non-blocking)**: Since we're importing a function to check the type then use a switch-like statement to return the text, we could also just put this entire thing into a util fn if we use it in multiple places like: `getStatusText(trace.status)`. Also, it would be ideal to check equality against constants and avoid string matching.

Reply:
Where's a good spot for that? `ui/packages/components/src/types/functionRun.ts` doesn't seem right since the user-facing status text is copy and not a type